src/pages/MovieListPage/MoviesListPage.jsx

```diff
@@ -0,0 +1,141 @@
+import "./MoviesListPage.css";
+import { useEffect, useState } from "react";
+import { SearchForm } from "../../entities/SearchForm/SearchForm";
+import { SelectTabs } from "../../entities/SelectTabs/SelectTabs";
+import { genresList } from "../../entities/SelectTabs/selectTabsMock";
+import { CustomButton } from "../../shared/ui/CustomButton/CustomButton";
+import { MovieDetails, MovieTitle, SortControl } from "../../entities";
+import { moviesSortOptions } from "../../entities/SortControl/sortControlMock";
+
+export const MovieListPage = () => {
+  const [movies, setMovies] = useState([]);
+  const [selectedMovie, setSelectedMovie] = useState(null);
+
+  const [searchQuery, setSearchQuery] = useState("");
+  const [sortCriterion, setSortCriterion] = useState(moviesSortOptions[0]);
+  const [selectedGenre, setSelectedGenre] = useState("ALL");
+
+  const handleSearch = (searchValue) => {
+    setSearchQuery(searchValue);
+  };
+
+  const handleChangeSelectedGenre = (event) => {
+    setSelectedGenre(event.target.value);
+  };
+
+  const handleChangeSortCriterion = (event) => {
+    setSortCriterion(event.target.value);
+  };
+
+  const handleMovieTitleClick = (movieProps) => {
+    setSelectedMovie(movieProps);
+  };
+
+  const handleMovieViewClosed = () => {
+    setSelectedMovie(null);
+  };
+
+  useEffect(() => {
+    const getMoviesData = async () => {
+      try {
+        const response = await fetch(
+          `http://localhost:4000?sortBy=${
+            sortCriterion.value
+          }?search=${searchQuery}?filter=${[selectedGenre]}`
+        );
+        const json = await response.json();
+        const data = JSON.stringify(json);
```

Comment:
also possible to short the code 
```suggestion
        const data = JSON.stringify(await response.json());
```

Reply:
done